Tests/TestLeafAuthData1.txt

```diff
@@ -0,0 +1 @@
+﻿{"vehicles":[{"uvi":"123456789012","brands":"N","modelcode":"17218","modelname":"LEAF","modelyear":"2018","extcolor":"","nickname":"LEA2018","pinAndPassword":null,"batteryRecords":{"operationResult":"START","lastUpdatedDateAndTime":"2018-09-17T01:57:34Z","batteryStatus":{"batteryChargingStatus":"YES","batteryCapacity":100,"batteryRemainingAmount":74,"soc":{"value":74}},"pluginState":"CONNECTED","cruisingRangeAcOn":186000.0,"cruisingRangeAcOff":188000.0,"timeRequired":null,"timeRequired200":null,"timeRequired200_6kW":{"hourRequiredToFull":3,"minutesRequiredToFull":0},"notificationDateAndTime":null},"interiorTempRecords":{"operationResult":null,"operationDateAndTime":null,"notificationDateAndTime":null,"inc_temp":"27.5"},"subcriptionList":[{"id":"1-2C34S3B","renewalDate":"2018-08-12-05:00","purchaseDate":"2018-04-16T23:00:26-05:00","subscriptionStartDate":"2018-04-12-05:00","subscriptionEndDate":"2018-10-16-05:00","subscriptionRemainingDays":"11","product":{"productID":null,"productName":"NissanConnect Services Leaf Premium - Trial","productCode":"NSNLFPREMTRIAL","marketingName":"NissanConnect Services Premium","productLine":"NSNPREM","services":["Automatic Collision Notification","Emergency Call","Roadside Assistance","Stolen Vehicle Locator","Alarm Notification","Maintenance Alert","Remote Door Lock","Remote Door Unlock","Remote Horn & Lights","Boundary Alert","Valet Alert","Curfew Alert","Speed Alert","Scheduled Maintenance Notification"]}},{"id":"1-2C25S2N","renewalDate":"2021-04-12-05:00","purchaseDate":"2018-04-12T23:00:26-05:00","subscriptionStartDate":"2018-04-23-05:00","subscriptionEndDate":"2021-04-23-05:00","subscriptionRemainingDays":"923","product":{"productID":null,"productName":"NissanConnect Services Leaf Select - Trial","productCode":"NSNLFSELECTTRIAL","marketingName":"NissanConnect Services Select","productLine":"NSNSELECT","services":["Maintenance Alert","Remote Door Lock","Remote Door Unlock","Scheduled Maintenance Notification"]}}],"genType":null,"deviceId":null,"svlStatus":"ACTIVE","accountId":null,"contactId":null,"subscriberType":null,"subscriptionId":null,"firstName":null,"lastName":null,"rdlRequestId":null,"rdulRequestId":null,"accountStatus":null,"productSpecificationCode":null,"remoteStartReqId":null,"remoteStopReqId":null,"enrollmentCompleted":true}],"authenticationInfo":{"pinConfigured":true,"securityQuestions":[{"questionId":"1"}]},"displayMessage":null,"displayNoTemperatureMessage":null,"accountID":"niscust:nis:123456","authToken":"somebogusvalue="}
```

Comment:
should this be a json file?

Reply:
The idea was to capture the JSON coming back from the web.  Sure, we can call that a .JSON file instead.